src/core/webidl.js

```diff
@@ -817,9 +817,13 @@ function findDfn(parent, name, definitionMap, type, idlElem) {
       name &&
       idlElem.classList.contains("no-link-warnings") === false;
     if (showWarnings) {
-      var msg = `No \`<dfn>\` for ${type} \`${originalName}\`${originalParent
-        ? " in `" + originalParent + "`"
-        : ""}`;
+      let msg = `No \`<dfn>\` for ${type} `;
```

Comment:
So, would really like this to be like:

```
const thingName = type === "operation" ? `${originalName}()` : originalName;
`Missing \`<dfn>\` for \`${originalParent}`\'s ${thingName} ${type}`;
```

Reply:
why didn't I think like that?!

Reply:
Is `originalParent` always present? Or I need to handle it?